tfjs-backend-webgpu/src/kernels/NonMaxSuppressionV3.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import {backend_util, KernelConfig, NonMaxSuppressionV3, NonMaxSuppressionV3Attrs, NonMaxSuppressionV3Inputs} from '@tensorflow/tfjs-core';
+import {WebGPUBackend} from '../backend_webgpu';
+export type TypedArray = Float32Array|Int32Array|Uint8Array;
```

Comment:
Why define `TypedArray` here? Can you import `TypedArray` from tfjs-core instead?

Reply:
Done here: https://github.com/tensorflow/tfjs/pull/2963/commits/756630081206adef8721d2b4055e6e0f113b7e59